docs/src/details/integration/chip/arm.rst

```diff
@@ -52,6 +52,13 @@ To add Arm2D to your project, follow these steps:
 Neon Acceleration
 -----------------
 
-Several Cortex-A microprocessors support the `Neon SIMD <https://www.arm.com/technologies/neon>`__ instruction set. LVGL has built-in support to improve the performance of software rendering by utilizing Neon instructions. To enable Neon acceleration, set ``LV_USE_DRAW_SW_ASM`` to ``LV_DRAW_SW_ASM_NEON`` in ``lv_conf.h``.
+Some ARM processors with the ARMv7 architecture and every ARM processor from the ARMv8 architecture support the `Neon SIMD <https://www.arm.com/technologies/neon>`__ instruction set.
```

Comment:
I think we should specify here, because not all ARMv7 processors are the A profile (Cortex-A), but M profile (Cortex-M), thats why the previous documentation entry mentioned Cortex-A instead of the ISA version, the same applies to ARMv8, not all processors that has this ISA support neon, the M profiles still don't support, they have theyr own vector extensions called MVE.